exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,16 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+function whoIsDaniel(danielsRole){
+  if (danielsRole === "mentor"){
+    return "Hi, I'm Daniel, I'm a mentor."
```

Comment:
This looks good, although you might consider using the name variable here, rather than repeating the name Daniel.